src/ops.rs

```diff
@@ -0,0 +1,257 @@
+use crate::{CResult, Config, NotifyGuard, SQ, clients, dyn_connect, static_config, Request};
```

Comment:
Can we rename this file? Maybe `put_get.rs` or something?

Reply:
What about `basic_ops.rs` (ops == operations) ? `put_get.rs` does not cover the delete op which is in the same file

Reply:
Maybe, it still seems a bit vague. How about `crud_ops.rs`? It doesn't perfectly match the cloud APIs but it gets the point across. Or move the delete into its own file even though it is small? Up to you

Reply:
Changed it  to `crud_ops.rs`